### ISSUE TITLE
pgClient.release() insufficient; use pgClient.end()

### ISSUE THREAD

**Issue by benjie** [calebmer/postgraphql#171]
When creating a postgraphql schema, when passed a `DATABASE_URL`, if you use `pgClient.release()` as done here:

https://github.com/calebmer/postgraphql/blob/ad8dd3f9a142f4cd776cf9abb799d1b2f3bf8162/src/postgraphql/schema/createPostGraphQLSchema.ts#L46

then the connection to postgres persists and the node process won't exit. Changing this to `pgClient.end()` solves the issue.

<details>
  <summary>e.g.</summary>
```js
import {createPostGraphQLSchema} from 'postgraphql';

const Schema = await createPostGraphQLSchema(process.env.DATABASE_URL, 'schema_data', {});
```
</details>

**Comment by calebmer** [comment]
Thanks for reporting, see https://github.com/calebmer/postgraphql/pull/173 :+1:

**Comment by calebmer** [comment]
Fixed and released in `v2.1.1` 💯
